.golangci.yml

```diff
@@ -51,6 +51,10 @@ linters:
         - appendAssign
         - exitAfterDefer
         - ifElseChain
+        - deprecatedComment
+    govet:
+      disable:
+        - buildtag
```

Comment:
Why disable it

Reply:
Seems you have already removed it?

Reply:
Yes, after removing, the ci is still normal.